js/main.js

```diff
@@ -114,61 +140,60 @@
 
       }
     ])
-    .service('true', ['$q',
-      function (q) {
-        return {
-          'get': function () {
-            var deferred = q.defer();
-
-            deferred.resolve(true);
+    .service('mockInterruptService', ['$q', '$log',
+      function (q, log) {
 
-            return deferred.promise;
-          }
+        function promise(value) {
+          var deferred = q.defer();
+          deferred.resolve(value);
+          return deferred.promise;
         }
-      }
-    ])
-    .service('false', ['$q',
-      function (q) {
+
         return {
-          'get': function () {
-            var deferred = q.defer();
 
-            deferred.resolve(false);
+          isInterruptRequired: function (interruptType) {
+            if (location.search.indexOf(interruptType) >= 0) {
+              return promise(true);
+            } else {
+              return promise(false);
+            }
+          },
 
-            return deferred.promise;
-          }
-        }
-      }
-    ])
-    .service('piuInterrupt', ['EasyXdm',
-      function (easyXdm) {
-        return {
-          'get': function (scope) {
-            return easyXdm.fetch(scope, '/wsapi/rest/staffwebinterruptrequired?popuptype=piu');
+          updateAgreementStatus: function(interruptType, status) {
+            log.info("updating agreement status: ", interruptType, status);
```

Comment:
This is in the `mockInterruptService`, which isn't used by end users at all. I only used it for testing.

If this project had unit tests set up, it would live in the the test source instead of the live source.